modules/azure-service-principal/main.tf

```diff
@@ -1,8 +1,6 @@
 resource "azuread_application" "adls-gen2-application" {
-  name                       = var.application_name
-  available_to_other_tenants = false
-  oauth2_allow_implicit_flow = true
-  type                       = "native"
+  display_name     = var.application_name
+  sign_in_audience = "AzureADMyOrg"
```

Comment:
Any use case for setting this to `AzureADMultipleOrgs`?

Reply:
I don't think so, I don't see a reason to allow access to tenants outside of the org we're deploying in